src/pudl/metadata/resources/ferc1_eia_record_linkage.py

```diff
@@ -166,9 +166,11 @@
         "field_namespace": "eia",
     },
     "out_pudl__yearly_assn_eia_ferc1_plant_parts": {
-        "description": """This table links power plant data reported in FERC Form 1 to related EIA data. It
+        "description": {
+            "additional_summary_text": "power plant data reported in FERC Form 1 and related EIA data.",
```

Comment:
non-blocking. the thing this header misses is that this table links whatever granular portion of a plant FERC1 reports with its corresponding portion of a plant from EIA. that why we call it plant "parts". FERC1 is all heterogeneous reporting units and generators and clumps of a plant that came online in 1980 vs another clump that came online in 2010.  So we had to make an eia plant parts table (with a bunch of possible combinations of generators) to link to FERC1.

imho this is hard to explain succinctly so i think this is fine as is but you seem to be good with words so if you can imagine a succinct way of saying this is connecting plant parts together that'd be swell